### DISCUSSION TITLE
Format code block content

### DISCUSSION THREAD

**Discussion by bestpika** [alagrede/znote-app#77] [Ideas]
Is it possible to add the feature to format the code block content?
The formatting rules then allow the user to define it in a code block (using ESLint).

Like this.
![image](https://user-images.githubusercontent.com/5698157/213410143-50d85dee-9e6f-4ed0-8fe8-f922275ea4f2.png)

**Comment by alagrede** [comment]
Hi @bestpika,
Thank you for your suggestion. I add it in my roadmap. Maybe I could start adding a quick option to format code with predefined options.

**Comment by alagrede** [comment]
Hi 👋 @bestpika,
In the last version (_2.1.0_), I added especially for you a code formatting option accessible on right click to format the current block or the selected code 😊
